tests/schema/test_private_field.py

```diff
@@ -23,6 +23,25 @@ class Query:
     assert instance.age == 22
 
 
+def test_private_field_set_through_type_decorator():
+    @strawberry.type(private_fields=["age"])
+    class Query:
+        name: str
+        age: int
+
+    definition = Query._type_definition
+
+    assert definition.name == "Query"
+    assert len(definition.fields) == 1
+
+    assert definition.fields[0].name == "name"
+    assert definition.fields[0].type == str
```

Comment:
I would add a check here that verifies that `age` is _not_ one of the fields on the `TypeDefinition`